src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -18,15 +18,13 @@
 public class ViewCommand extends Command {
     public static final String COMMAND_WORD = "view";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Display all relevant contacts under a task "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Display all contacts assigned the task identified "
```

Comment:
Perhaps it should be "assigned to the task identified"? Maybe missing a "to"

Reply:
👌